tests/integration/retry.py

```diff
@@ -0,0 +1,52 @@
+"""async implementation of a retry decorator algorithm with similar api to retry
+
+   Perhaps one day this package will have retry for async methods
```

Comment:
it looks like Tenacity supports [retries for async methods](https://tenacity.readthedocs.io/en/latest/index.html?highlight=async#async-and-retry)

Reply:
thanks stone... i'll go look at that